### PR TITLE
Zero-pad string representation of addresses

### LINKED ISSUE #1131: Zero-padding of Address#toString()
### Instructions

When you execute Address#toString() in Cadence, it does not return a zero-padded result.
If the length of Address is fixed, I think it should be zero-padded result.

### Problem

When I used Cadence Script to get metadata containing address information as `{String:String}` type value and compared it with a string, I could not check that it was the same address.

### Steps to Reproduce

CADENCE: V0.18.0

```
  let addr: Address = 0x004b19d1e7c70d25
  log(addr.toString())
  // -> "0x4b19d1e7c70d25"
```

### PR DESCRIPTION
**Pull request by turbolent**
Closes #1131 

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

**Review by jordanschalm — APPROVED** [review]
🎸

File: runtime/sema/check_composite_declaration.go [hunk 1]
```diff
@@ -1229,8 +1229,21 @@ func (checker *Checker) checkTypeRequirement(
 	for _, nestedCompositeDeclaration := range containerDeclaration.Members.Composites() {
 		nestedCompositeIdentifier := nestedCompositeDeclaration.Identifier.Identifier
 		if nestedCompositeIdentifier == declaredCompositeType.Identifier {
+			// If we detected a second nested composite declaration with the same identifier,
```
Comment (SupunS) [inline runtime/sema/check_composite_declaration.go:1232]: I think this is not related to `address` fix. Is it intentionally included?

Comment (turbolent) [reply runtime/sema/check_composite_declaration.go:1232]: Argh, good flag! I accidentally committed this in the wrong PR, removing

**Review by SupunS — APPROVED** [review]
Looks good! Thank you!

**Pull request merged**